### DISCUSSION TITLE
Managing bot equipment in multiplayer lobbies

### DISCUSSION THREAD

**Discussion by Quartofel** [FakeFishGames/Barotrauma#7474] [Ideas and suggestions]
As of right now, the only way to change equipment of a bot in MP is to beat the poor folks into submission and then either cuff them or just switching their gear.
It's not only inconvinient, but also quite immersion breaking, and fairly important gameplay-wise (for instance, having gunners wear bandoliers and gunner's helmet give them a _significant_ performance boost). Also, bots start to retaliate after couple of inventory management stunbaton sessions.

Potential implementations:
-let host switch to bots so they can manually equip gear on bots;
-allow to manipulate bot's equipment when H-holding them;
-give captain ability to order bots to equip items;
-give captain bot inventory management menu;

**Comment by MateusJK15** [comment]
Would be awesome to give an order like "Prepare for combat" and see a bot search around the submarine and equip itself with body armor and a gun.

**Comment by mLuby** [comment]
I like `allow manipulating bot's equipment when H-holding them` best. Your crew bots would let you do that whenever, hostiles bots never would—you have to stun them, and unarmed neutral bots like station NPCs wouldn't unless you're aiming a weapon at them but then they might call security on you.
